proposal/color-4-new-spaces.md

```diff
@@ -1161,7 +1213,13 @@ input colors.
 * If `weight` is null, set `weight` to `0.5`.
 
 * Otherwise, set `weight` to the result of [percent-converting] `weight` with a
-  max of 1, and then clamping the value between 0 and 1 (inclusive).
+  max of 1.
+
+* If `weight` is greater than 1 or less than 0, throw an error.
```

Comment:
Probably a good idea to explicitly write `weight > 1` and `weight < 0`, so that it's clear that this uses Sass's fuzzy comparison logic.